gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -1622,29 +1655,93 @@ gst_tensor_converter_parse_caps (GstTensorConverter * self,
 
   /** set the number of frames in dimension */
   if (frames_dim >= 0) {
-    config.info.dimension[frames_dim] = self->frames_per_tensor;
+    config.info.info[0].dimension[frames_dim] = self->frames_per_tensor;
   }
 
-  if (!gst_tensor_config_validate (&config)) {
+  if (!gst_tensors_config_validate (&config)) {
     /** not fully configured */
-    GST_ERROR_OBJECT (self, "Failed to configure tensor info.\n");
+    GST_ERROR_OBJECT (self, "Failed to configure tensors info.\n");
     return FALSE;
   }
-
   if (gst_tensor_info_validate (&self->tensor_info)) {
     /** compare tensor info */
-    if (!gst_tensor_info_is_equal (&self->tensor_info, &config.info)) {
+    if (!gst_tensor_info_is_equal (&self->tensor_info, &config.info.info[0])) {
       GST_ERROR_OBJECT (self, "Failed, mismatched tensor info.\n");
       return FALSE;
     }
   }
 
   self->in_media_type = in_type;
-  self->tensor_configured = TRUE;
-  self->tensor_config = config;
+  self->tensors_configured = TRUE;
+  self->tensors_config = config;
+  return TRUE;
+}
+
+/**
+ * @brief Update pas caps from tensors config
+ */
+static gboolean
+gst_tensor_converter_update_caps (GstTensorConverter * self,
+    GstPad * pad, GstTensorsConfig * config)
+{
+  GstStructure *st;
+  GstCaps *curr_caps, *out_caps = NULL;
+
+  if (config->info.num_tensors == 1) {
+    GstCaps *peer_caps = gst_pad_peer_query_caps (pad, NULL);
+    silent_debug_caps (peer_caps, "********sink event peer caps********");
```

Comment:
plz check additional log message for peer caps

Reply:
Ooops, Thanks. This line is removed because it's added for debugging.